src/routes/solid-router/advanced-concepts/preloading.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: Preloading
+---
+
+Anchors in Solid Router will preload routes by default on link hover/focus to improve perceived performance.
+
+To enhance preloading, you can define the `preload` function on your route definition.
+When on a [SolidStart](/solid-start) application, this function can also run on the server during the initial page load to start fetching data before rendering. When in a Single-Page Application (SPA), it will load the route's component and its `preload` function when the user hovers or focuses on a link.
+
+| user action | route behavior                          |
+| ----------- | -------------------------------------- |
+| hover   | with a 300ms delay to avoid excessive preloading  |
```

Comment:
Curious where you got the 300ms from. I can only see a 20ms timeout [here](https://github.com/solidjs/solid-router/blob/main/src/data/events.ts#L94)